src/components/common/Notifications/index.tsx

```diff
@@ -57,7 +79,7 @@ const Notifications = (): ReactElement | null => {
     <div className={css.container}>
       {visible.map((item) => (
         <div className={css.row} key={item.id}>
-          <Toast message={item.message} severity={item.variant || 'info'} onClose={() => handleClose(item)} />
+          <Toast {...item} onClose={() => handleClose(item)} />
```

Comment:
Wouldn't this spread into `variant` instead of `severity`? And do you remember why we had the fallback to "info" before?

Reply:
`<Toast>` now accepts `props.variant`, which it passes to the `severity` prop of the `<Alert>`. `variant` is a required argument of `showNotification`.